op-service/signer/transaction_args.go

```diff
@@ -28,16 +35,20 @@ type TransactionArgs struct {
 
 	AccessList *types.AccessList `json:"accessList,omitempty"`
 	ChainID    *hexutil.Big      `json:"chainId,omitempty"`
+
+	// Custom extension for EIP-4844 support
+	BlobVersionedHashes []common.Hash `json:"blobVersionedHashes,omitempty"`
+	BlobFeeCap          *hexutil.Big  `json:"blobFeeCap,omitempty"`
```

Comment:
Not included in op-geth internal version yet, but in geth v1.13.11 they added it. And called the blob fee cap `maxFeePerBlobGas` in JSON. Will update the PR